main.css

```diff
@@ -66,52 +82,64 @@ body {
     align-items: center;
     justify-content: center;
     flex-direction: column;
-    gap: 80px;
-    padding: 100px 0px 100px 0px;
-    font-size: 100px;
-    font-weight: 550;
+    gap: 30px;
+    padding: 10px 0px 50px 0px;
+}
+
+.let-ride__container {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    gap: 30px;
+}
+
+.let-ride__container_small {
+    text-align: center;
 }
 
 .let-ride__header {
-    font-size: 100px;
-    font-weight: 550;
+    font-size: 48px;
+    font-weight: 600;
 }
 
 .let-ride__buttons {
     display: flex;
     justify-content: center;
     align-items: center;
     flex-direction: column;
-    gap: 40px;
+    gap: 20px;
 }
 
-.let-ride__apply-button {
+.let-ride__button {
     line-height: 2.5;
-    padding: 0px 70px 0px 70px;
-    font-size: 40px;
+    padding: 0px 7px 0px 7px;
+    font-size: 17px;
+    width: 150px;
     text-align: center;
     color: #FFFFFF;
-    border-radius: 200px;
+    border-radius: 20px;
     background-color: #523be4;
-    font-weight: 600;
+    font-weight: 60;
 }
 
-.let-ride__sign-button {
+.let-ride__button--empty {
     line-height: 2.5;
-    padding: 0px 70px 0px 70px;
-    font-size: 40px;
-    text-align: center;
+    padding: 0px 7px 0px 7px;
+    font-size: 17px;
+    width: 150px;
     color: #000;
-    border-radius: 200px;
+    text-align: center;
+    border-radius: 20px;
     background-color: #FFFFFF;
-    border: 3px solid grey;
-    position: relative;
+    border: 1px solid grey;
     font-weight: 600;
 }
 
 .let-ride__image{
-    width: 900px;
-    height: 500px;
+    width: 92%;
```

Comment:
hmm, can we avoid doing this somehow? looks wonky

Reply:
+